rs/registry/canister/src/get_node_providers_monthly_xdr_rewards.rs

```diff
@@ -35,16 +38,8 @@ impl Registry {
         // We want to have as many independent NPs as possible for the given reward budget.
         let mut np_coefficients: HashMap<String, f64> = HashMap::new();
 
-        for (key, values) in self.store.iter() {
-            if key.starts_with(NODE_OPERATOR_RECORD_KEY_PREFIX.as_bytes()) {
-                let value = values.back().unwrap();
-                if value.deletion_marker {
-                    continue;
-                }
-                let node_operator = NodeOperatorRecord::decode(value.value.as_slice()).unwrap();
-                let node_operator_id = PrincipalId::try_from(
-                    &node_operator.node_operator_principal_id,
-                )
+        for (key, node_operator) in get_key_family_iter(&self, NODE_OPERATOR_RECORD_KEY_PREFIX) {
```

Comment:
Note for reviewer: These are the relevant changes, everything else is whitespace